modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -4533,6 +4536,7 @@ cross-references=Cross References
 cross-sell=Cross-Sell
 crypto-hash-provider-configuration-name=Hash Provider Configuration Name
 crypto-hash-provider-configuration-name-description=Set hash provider configuration name.
+cs-diagram-entry-id=Represents the ID of the diagram entry
```

Comment:
Same here

Reply:
This is not something we can show to the citizen dev, please change it with something readable.